variants/twatch/pins_arduino.h

```diff
@@ -24,8 +24,8 @@
 
 static const uint8_t KEY_BUILTIN = 36;
 //Serial1 Already assigned to GPS LORA
-static const uint8_t TX = 33;
-static const uint8_t RX = 34;
+static const uint8_t TX1 = 33;
+static const uint8_t RX1 = 34;
```

Comment:
not rename :) use `#define` as in my example. Reason is that the defines will overwrite the default pins, but a declaration will cause issues:  https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/HardwareSerial.cpp#L6-L23

Reply:
It has been modified. Thanks